### PR TITLE
Module spp audit log

### PR DESCRIPTION
**Pull request by reichie020212**

### CODE REVIEW

**Review by kneckinator — CHANGES_REQUESTED** [review]
Can you please add tests to the new modules @reichie020212 ?
Thanks!

File: spp_audit_log/models/spp_audit_rule.py [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+            return None, []
+
+        # Initialize variable to be used in while loop
+        current_rule_id = self
+        parent_rule_id = self.parent_id
+        currect_model_records = {"model": self.model_id.model, "ids": res_ids}
+
+        # get the model name and ids of the most parent rule
+        # loop will break if a rule doesn't have parent rule
+        while parent_rule_id:
```
Comment (nhatnm0612) [inline spp_audit_log/models/spp_audit_rule.py]: @reichie020212 I think this logic will be improved if we are using searching domain operator `parent_of` or `child_of`

**Comment by reichie020212** [comment]
> Can you please add tests to the new modules @reichie020212 ? Thanks!

done adding tests

**Comment by gonzalesedwin1123** [comment]
Merged for transfer to mono-repo.

**Pull request merged**